features/admin.core.v1/package.json

```diff
@@ -74,17 +74,18 @@
         "@wso2is/admin.webhooks.v1": "^1.1.0",
         "@wso2is/admin.wsfed-configuration.v1": "^2.25.205",
         "@wso2is/common.ui.v1": "workspace:^",
+        "@wso2is/common.workflow-approvals.v1": "^1.0.9",
```

Comment:
Is this intentional?

Reply:
hey @pavinduLakshan this is because of using pnpm add dayjs command
order of few dependencies changed but not removed in this file it was removed in line 55 and added to line 42

Reply:
Ack, seems it's used in the core.v1 module. So no worries :)